### ISSUE TITLE
react-dates css classes being removed after production build

### ISSUE THREAD

**Issue by stevenfitzpatrick** [airbnb/react-dates#1210]
Running into an issue with react dates where the class names seem to be removed in webpack production build.

i.e. inspecting the styles locally of the date picker it looks something like

`SingleDatePickerInput_calendarIcon_15s124x`

But then when run in production it just looks like

`_1i1103bk`

Only getting this issue when using the theming set up and Themedstylesheet libraries

```
import 'react-dates/lib/css/_datepicker.css';
import ThemedStyleSheet from 'react-with-styles/lib/ThemedStyleSheet'
import aphroditeInterface from 'react-with-styles-interface-aphrodite'
import DefaultTheme from 'react-dates/lib/theme/DefaultTheme'
import { colors } from '@swrve/tailwind-config'
```

Is there anyway to change this behaviour in the library or webpack to not shorten the classnames produced ?

Version
"react-dates": "^16.7.0",

**Comment by majapw** [comment]
@lencioni I think this is an Aphrodite performance improvement for production specifically.

If you need deterministic classnames for overriding or something similar, it'd probably be better to use the css interface for react-with-styles instead of the aphrodite one. @stevenfitzpatrick can you share a bit more on what you need the classnames for?

**Comment by lencioni** [comment]
Yes, Maja is correct. This is what Aphrodite does for class names in production. I believe this is the relevant code, if you are curious: https://github.com/Khan/aphrodite/blob/62ee2f0a39f10be75322dc0f77d1346e9731ffbf/src/inject.js#L307-L313

**Comment by leonardolessa** [comment]
I have an issue with the same element, but I guess the origin of the problem is that we don't have a way to override the spacing of the inputIcon (calendarIcon) using the registerTheme.
